### PR TITLE
Remove time and seeds from MESMER bundle

### PR DESCRIPTION
**Pull request by znicholls**
Hopefully fairly straight-forward, I think I did the right changes...

### CODE REVIEW

**Review by mathause — APPROVED** [review]
Looks good.

**Review by leabeusch — APPROVED** [review]
Thanks @znicholls! Looks good to me too (just suggest to remove a few more docstring lines that aren't necessary anymore). Can you please also make sure that the changes you made to the mesmer bundle content here are propagated into what you started in mesmer-openscmrunner? 

This PR btw closes #25.

**Comment by leabeusch** [comment]
One thing I only noticed after approving this PR: if I run `isort` on these scripts, `calibrate_mesmer.py` and `test_draw_realisations_from_bundle.py` both get changed. Do we not check if `isort` is run in our tests at the moment? Can we add it? + Can you please still run it before merging this PR @znicholls? Or is there another reason I obtain changes when running `isort` here? (maybe @mathause also has an answer about this for me?)

**Comment by mathause** [comment]
See #69.

**Comment by leabeusch** [comment]
Ok, sadly I had to learn that just because I update master doesn't mean, the tests on this PR are getting re-run -> this PR still claims to pass all the tests although it wouldn't anymore since `isort` is now part of the tests -> can you please quickly run `isort` and add the commit for it to this PR @znicholls  before we merge into master? (The alternative of creating my own branch that applies `isort` and merging it into yours seems more cumbersome)

**Comment by mathause** [comment]
This PR is not updated when master is changed. @znicholls will merge master and then the test will be re-run and should also pass because this also picks up the changes in the imports in #69

**Comment by leabeusch** [comment]
Hmmm, but when I ran `isort` on my computer on this branch before, it also picked up a line in `test_draw_realisations_from_bundle.py` (just some missing or extra empty line; don't remember which anymore) which wasn't part of your isort_CI PR -> I don't think the changes of this PR are fully isort compatible yet? Unless the test just runs isort and commits the changes during merging. But that's not the case, no?

**Comment by znicholls** [comment]
I’ll rebase this branch before merging

File: tests/integration/test_calibrate_mesmer.py [hunk 1]
```diff
@@ -52,28 +53,32 @@ def test_calibrate_mesmer(test_data_root_dir, tmpdir, update_expected_files):
...
+            for k in res:
+                res_val = res[k]
+                exp_val = exp[k]
 
-    _check_dict(res, exp)
+                assert type(res_val) == type(exp_val)
+                if isinstance(res_val, dict):
+                    _check_dict(res_val, exp_val)
+                elif isinstance(res_val, sklearn.linear_model.LinearRegression):
+                    # not sure if there's a better way to test this...
```
[the remark this answers is not in the document]
Comment (znicholls) [reply tests/integration/test_calibrate_mesmer.py:75]: Yep, I wasn't sure if the models were defined by more than their coefficients. Sounds good

**Comment by znicholls** [comment]
> Can you please also make sure that the changes you made to the mesmer bundle content here are propagated into what you started in mesmer-openscmrunner?

I'll do that once we start versioning MESMER properly (no point me chasing every tiny change we make right now). Perhaps #31 could be addressed next (a 20 minute job for @mathause ?).

**Pull request merged**